### ISSUE TITLE
Legend Name Different in Plotly

### ISSUE THREAD

**Issue by brianboonstra** [ropensci/plotly#188]
If I create the ggplot below and public using the API, I expect to see a label for the color legend, but I don't get one.

    ggplot(aes(x=returns.both$rho,y=returns.both$profit,color=returns.both$BetaAdjusted),data=returns.both)
           + geom_point(size=5, alpha=0.45)
           + labs(title="CDS Pairs Trading Profits (12 Weeks)", x="Rho", y="Profit", colour="Beta Adjusted") 
           + geom_hline(y=0) + geom_vline(x=0)

Expected:
![cdspairs](https://cloud.githubusercontent.com/assets/55118/6657658/67623782-cb20-11e4-81a0-1ffb7973d3f0.png)

Obtained:
![](https://plot.ly/~brianboonstra/115.png)
https://plot.ly/~brianboonstra/115

**Comment by mkcor** [comment]
This issue might be a duplicate of https://github.com/ropensci/plotly/issues/159.

**Comment by 13bzhang** [comment]
@mkcor The problem seems to have gone away. 

See https://plot.ly/~13bzhang/982/cds-pairs-trading-profits-12-weeks/

![982](https://cloud.githubusercontent.com/assets/6131842/8366366/75255dee-1b62-11e5-99de-7d8448871b25.png)

```r
rho <- round(rnorm(25, 0, 5))
profit <- 0.5 + 0.3 * rho + rnorm(25, 0, 1)
BetaAdjusted <- factor(c(rep(TRUE, 15), rep(FALSE, 10)))
returns.both <- data.frame(rho, profit, BetaAdjusted)
p <- ggplot(aes(x=rho, y=profit,
           color=BetaAdjusted),data=returns.both) + 
  geom_point(size=5, alpha=0.45) +
  labs(title="CDS Pairs Trading Profits (12 Weeks)", 
       x="Rho", y="Profit", colour="Beta Adjusted") +
  geom_hline(y=0) + geom_vline(x=0)
```

**Comment by 13bzhang** [comment]
Ok, I noticed a new problem. The name of the legend doesn't have the space between "Beta" and "Adjusted."

ggplot2:

![118](https://cloud.githubusercontent.com/assets/6131842/8368799/37b727ce-1b7d-11e5-82e2-ce7a4e57d2c7.png)

I'll change the name of this issue to "Legend Name Different in Plotly".

**Comment by cpsievert** [comment]
This should be fixed via 6053d9b, please reopen if it isn't.
